ios/ZulipMobile/ZulipMobile.entitlements

```diff
@@ -4,5 +4,9 @@
 <dict>
 	<key>aps-environment</key>
 	<string>development</string>
+	<key>com.apple.developer.applesignin</key>
+	<array>
+		<string>Default</string>
+	</array>
```

Comment:
What's this bit mean? (Did it come from some instructions somewhere?)

Reply:
Yeah, it's step one of https://github.com/expo/expo/tree/1c5eb9818/packages/expo-apple-authentication#setup-ios-project, adding the "Sign in with Apple" entitlement.

Reply:
Great, and I see you've added this info to the relevant commit message.